integration-tests/helpers/index.js

```diff
@@ -218,18 +216,20 @@ async function execHelper (command, options) {
   /* eslint-disable no-console */
   try {
     console.log('Exec START: ', command)
-    await exec(command, options)
+    await execSync(command, options)
```

Comment:
How does `execSync` reduce flakiness?

Also `execSync` is synchronous, by definition, so there's no need to `await` it.